### PR TITLE
Patcher v2

### PR DESCRIPTION
**Pull request by lubiarz-oliwia**

### CODE REVIEW

File: _data/cis-how-it-works.yml [hunk 1]
```diff
@@ -39,8 +39,16 @@
...
+      Our CLI tool automates the entire process of updating code dependencies, allowing you to stay up-to-date without any hassle. 
+      <a href="/patcher" target="_blank">Check out how simple it is.</a>
+    </p>
+
 - title: Have another compliance objective?
   description: |
     <p>
       Are you looking to achieve compliance with PCI DSS, HIPAA, NIST, or another standard? <a href="/contact">Contact us</a> to discuss your project!
-    </p>
+    </p>
```
Comment (klijakub) [inline _data/cis-how-it-works.yml]: No new line at the end

File: _data/patcher-how-it-works-march.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    with-patcher: Run the Patcher CLI to automatically discover dependencies in your code, the versions they are at, and if new versions are available.
+    without-patcher: You have to manually scan your code for dependencies, manually track which versions they are at, and manually look up if new versions are available.
+    asciinema-id: patcher-player-cast-1
+    owner: user
+  - title: Hassle-free updates with step-by-step instructions and automatic patching
+    step-image: 2
+    with-patcher: Choose the desired module and its corresponding version for an update. Patcher automatically modifies your code, applying necessary patches. You will be guided through any necessary manual changes.
+    without-patcher: Update to new versions manually. If the new version is backward incompatible, go searching for a migration guide, and follow the steps to update your code manually. Reply
+    asciinema-id: patcher-player-cast-2
+    owner: user
```
Comment (klijakub) [inline _data/patcher-how-it-works-march.yml]: No newline at the end

File: _data/patcher.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  planned: Windows
+- name: Update method
+  beta: Update the folders you want with the CLI
+  planned: Set up automatic updates with the web UI
+- name: Availability
+  beta: Part of the Gruntwork Subscription
+  planned: Standalone product
+- name: Supported modules
+  beta: Gruntwork authored modules
+  planned: Gruntwork modules and your own modules
```
Comment (klijakub) [inline _data/patcher.yml]: No newline at the end

File: _data/sitemap.yml [hunk 4]
```diff
@@ -82,4 +85,4 @@
       url: "https://github.com/gruntwork-io/knowledge-base/discussions"
 
     - title: Gruntwork Store
-      url: "https://store.gruntwork.io/"
+      url: "https://store.gruntwork.io/"
```
Comment (klijakub) [inline _data/sitemap.yml]: No newline at the end

File: pages/early-access/_form.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    </div>
+  </div>
+  <div class="text-center">
+    <br />
+    <button type="submit" id="submit-button" class="btn btn-primary" ga-on="click" ga-event-category="early-access"
+      ga-event-action="submit" href="/great-news">
+      Submit
+    </button>
+  </div>
+</form>
```
Comment (klijakub) [inline pages/early-access/_form.html]: No newline

File: pages/early-access/_hero.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+            margin-left: 12px;
+          " class="label-beta label-small">
+            BETA
+          </div>
+        </label>
+      </h1>
+      <p style="margin-bottom:4px" class="lead">Take the first step towards effortless code</p>
+    </div>
+  </div>
+</div>
```
Comment (klijakub) [inline pages/early-access/_hero.html]: No newline

File: pages/early-access/index.html [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+      </div>
+      <div class="row row-form" style="display: flex; justify-content: center;">
+        <div class="col-xs-12">
+          {% include_relative _form.html %}
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (klijakub) [inline pages/early-access/index.html]: No newline

File: pages/pricing/_standard-pricing.html [hunk 8]
```diff
@@ -5,63 +5,51 @@ <h2 class="no-anchor margin-top-none">Standard</h2>
...
-          Annual contract, billed monthly.
-          <br>
-          View our
-          <span><a href="/terms/" title="Gruntwork Terms of Service" target="_blank">Terms of Service</a></span>.
-        </em>
-      </p>
-    </div>
   </div>
-</div>
+</div>
```
Comment (klijakub) [inline pages/pricing/_standard-pricing.html]: No newline

File: pages/patcher/index.html [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12">
+          {% include_relative _how-it-works.html %} {% include_relative
+          _features.html %} {% include_relative _patcher-table.html %}
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (klijakub) [inline pages/patcher/index.html]: No newline

File: pages/patcher/_how-it-works.html [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+                        <img class="patcher-icon" src="/assets/img/patcher/close-icon@3x.png" />
+                        <h4>Without Patcher</h4>
+                    </div>
+                    <p>{{item.without-patcher}}</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    {% endfor %} {% endfor %}
+</div>
```
Comment (klijakub) [inline pages/patcher/_how-it-works.html]: No newline

File: pages/index/_learn-more.html [hunk 11]
```diff
@@ -144,3 +147,59 @@ <h2>Achieve compliance.</h2>
...
+        class="btn btn-primary"
+        role="button"
+        ga-on="click"
+        ga-event-category="home-learn-more"
+        ga-event-action="preview-training"
+        >See how Patcher works</a
+      >
+    </p>
+  </div>
+</div>
```
Comment (klijakub) [inline pages/index/_learn-more.html]: No newline

File: pages/index/_learn-more.html [hunk 12]
```diff
@@ -144,3 +147,59 @@ <h2>Achieve compliance.</h2>
...
+      and applies to all Gruntwork maintained modules.
+    </p>
+    <p>
+      <a
+        href="/patcher/"
+        class="btn btn-primary"
+        role="button"
+        ga-on="click"
+        ga-event-category="home-learn-more"
+        ga-event-action="preview-training"
```
Comment (klijakub) [inline pages/index/_learn-more.html]: Not sure about these ga-event tags 
Maybe should be different than here:
```
<a
        href="/training/"
        class="btn btn-primary"
        role="button"
        ga-on="click"
        ga-event-category="home-learn-more"
        ga-event-action="preview-training"
        >Preview Our Training</a
      >
```

File: pages/early-access/index.html [hunk 13]
```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Get early access to Patcher
+permalink: /early-access/
+slug: early-access
+custom_js:
+- contact
```
Comment (klijakub) [inline pages/early-access/index.html:7]: Check if we can use the same script from the contact, or if we need another one

File: pages/early-access/_form.html [hunk 14]
```diff
@@ -0,0 +1,34 @@
+<div id="error-message"></div>
+<form id="contact-form" method="post" target="_blank"
+  action="https://api.formbucket.com/f/buk_7iB8j7vEJPW9ad2ClJwFfm5M">
```
Comment (klijakub) [inline pages/early-access/_form.html:3]: Probably we have to ask for new formbucket link for this type of contact

**Review by klijakub — CHANGES_REQUESTED** [review]

File: assets/css/global.scss [hunk 15]
```diff
@@ -589,6 +610,13 @@ a.caret-right {
       margin-right: 0;
     }
   }
+  @media (min-width: 1150px) {
```
Comment (ArkuVonSymfon) [inline assets/css/global.scss:613]: Its necessary to use custom width? If not, please use scss variables

File: pages/compliance/_how-it-works.html [hunk 16]
```diff
@@ -1,17 +1,47 @@
...
+          BETA
+        </div>
+      </label>
+      {% endif %}
+    </h3>
+    {{ step.description }}
+  </div>
+</div>
 {% endfor %}
-
```
Comment (ArkuVonSymfon) [inline pages/compliance/_how-it-works.html:17]: end line

**Review by ArkuVonSymfon — CHANGES_REQUESTED** [review]

File: pages/early-access/index.html (hunk 13 above)
Comment (lubiarz-oliwia) [reply pages/early-access/index.html:7]: All forms use the same one, should be fine.

File: pages/early-access/_form.html (hunk 14 above)
Comment (lubiarz-oliwia) [reply pages/early-access/_form.html:3]: I already asked Jason.

File: assets/css/global.scss (hunk 15 above)
Comment (lubiarz-oliwia) [reply assets/css/global.scss:613]: Unfortunately, yes.

File: assets/css/pages/patcher.scss [hunk 17]
```diff
@@ -79,6 +84,7 @@
         }
 
         td {
+
```
Comment (ArkuVonSymfon) [inline assets/css/pages/patcher.scss]: unnecessary brake

File: assets/css/pages/patcher.scss [hunk 18]
```diff
@@ -71,6 +75,7 @@
 
         &:last-child {
           td {
+
```
Comment (ArkuVonSymfon) [inline assets/css/pages/patcher.scss]: unnecessary brake

File: assets/css/pages/patcher.scss [hunk 19]
```diff
@@ -62,6 +65,7 @@
       tr {
         &:first-child {
           td {
+
```
Comment (ArkuVonSymfon) [inline assets/css/pages/patcher.scss]: unnecessary brake

File: assets/css/pages/patcher.scss [hunk 20]
```diff
@@ -593,4 +592,28 @@
...
+    .close-popup-btn {
+      position: absolute;
+      bottom: 20px;
+      right: 20px;
+      padding: 0;
+      border: none;
+      cursor: pointer;
+    }
+  }
+}
```
Comment (ArkuVonSymfon) [inline assets/css/pages/patcher.scss:613]: check last line here

File: pages/patcher/_how-it-works.html [hunk 21]
```diff
@@ -43,3 +43,38 @@ <h4>Without Patcher</h4>
...
+  });
+
+  closeBtns.forEach((closeBtn, index) => {
+    if (window.innerWidth < screenMdMin) {
+      closeBtn.addEventListener("click", function () {
+        popups[index].classList.remove("popup-screen-active");
+      });
+    }
+  });
+</script>
```
Comment (ArkuVonSymfon) [inline pages/patcher/_how-it-works.html:80]: here too check last line

**Review by ArkuVonSymfon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**